src/sql/workbench/services/notebook/browser/models/clientSession.ts

```diff
@@ -332,7 +332,18 @@ export class ClientSession implements IClientSession {
 	 * this is a no-op, and resolves with `false`.
 	 */
 	restart(): Promise<boolean> {
```

Comment:
Is there any reason to have this return a boolean given that the public APIs don't seem to care about the return value?

Reply:
Yeah was thinking about this too. Lemme look at what changing this to void would mean re: other changes.